<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmbNucPinCellEditor</class>
 <widget class="QWidget" name="cmbNucPinCellEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PinCell Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="labelLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Origin:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string> X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="originY"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="originX"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pitch:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string> X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pitchX"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pitchY"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pitchZ"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pieces:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTableWidget" name="piecesTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTableWidget" name="layersTable">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="addLayerBeforeButton">
       <property name="text">
        <string>Add Before</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addLayerAfterButton">
       <property name="text">
        <string>Add After</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteLayerButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="cutAwayViewCheckBox">
     <property name="text">
      <string>Cut-away View</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>splitter</zorder>
  <zorder>addButton</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
